facerecognition.py

```diff
@@ -118,6 +119,7 @@ def Game():
                 )
             else:
                 dot.MoveRandomly(refreshTime)
+                print("Random")

```

Comment:
Let's try not to have debug code on main.

Reply:
Sounds good!

Reply:
It do be gone tho